packages/monaco-editor/scripts/addMonacoWebpackConfig.js

```diff
@@ -9,21 +10,56 @@ const path = require('path');
  * @param {import('webpack').Configuration & { [key: string]: any }} config - Webpack config to merge the
  * monaco-related config into. It must be an object (not an array or function). `config.entry`
  * also must be an object (not a string, array, or function).
- * @param {boolean} [includeAllLanguages] - If true, include all language contributions in the main
+ * @param {object} [options]
```

Comment:
this wont work probably as it doesn't mean what it looks like it mean. 

`object` in typescript  means non primitive type. What you need to use is `Object` in this case